### ISSUE TITLE
[BUG] The version of the package cannot be determined

### ISSUE THREAD

**Issue by pjhavariotis** [dannagle/PacketSender#376]
After installing the latest package via `winget`, the version of the package cannot be determined (Version: Unknown)
#### What OS and Version?
Windows: Windows.Desktop v10.0.19045.4412

#### Description of issue
Please check the following issue for more info:
https://github.com/microsoft/winget-pkgs/issues/158740

**Comment by Trenly** [comment]
For additional context - `portable` installers (loose binaries) do not need to write anything to the Registry.

The version that is currently used for WinGet appears to be an `inno` installer - which should write to the registry by default as long as the `AppVersion` is set in the inno setup file
https://jrsoftware.org/ishelp/index.php?topic=setup_appversion

**Comment by pjhavariotis** [comment]
> I was able to recreate the error.
> 
> ```
> This package's version number cannot be determined. To upgrade it anyway, add the argument --include-unknown to your previous command.
> ```
> 
> Registry entries is a new winget requirement. By design, Packet Sender has never modified the registry. Not touching the registry makes the portable versions a lot easier to maintain and upgrade.
> 
> I can make an exception within the installer itself to try to keep the portable version fully portable. It'd have to be on the next version.
> 
> To be clear, the suggested `--include-unknown` bypass is working for you, correct?

Correct. However, it would be nice to follow the `winget` requirements.

**Comment by dannagle** [comment]
This is now fixed in the development branch. The next release will require one more `--include-unknown`, and then it should work as expected after that. 

I'm leaving this ticket open as a reminder to doublecheck.
